client/src/containers/login/Login.js

```diff
@@ -39,62 +36,66 @@ function Login() {
       .catch((err) => setMessage(`${err.response.data.message.message}`));
   };
 
-   
   return (
-    <Grid
-    container
-    direction="column"
-    justify="center"
-    alignItems="center"
-  >
     <main className="main_login">
-      <form  data-testid="form-component" onSubmit={handleSubmit(onSubmit)}>
+      <div className='centrilized-content'>
+      <img src={secure} alt="secure"/>
+      <form
+        className="form-login"
+        data-testid="form-component"
+        onSubmit={handleSubmit(onSubmit)}
+      >
         <p>{message}</p>
-        <label  data-testid="test-label" htmlFor="email">
+        <div className='input-component'>
+        <label data-testid="test-label" htmlFor="email">
           E-mail
-        </label >
+        </label>
         <input
           data-testid="input-form-email"
           type="text"
           name="email"
           ref={register({ required: true })}
-
-       /*    color="white"
-           startAdornment={
-            <InputAdornment position="start">
-              <AccountCircle />
-            </InputAdornment>
-          }  */
         />
         {errors.email && "This field is required"}
+        </div>
+        <div className='input-component'>
 
-        <label  data-testid="test-label" htmlFor="password">
+        <label data-testid="test-label" htmlFor="password">
           Password
-        </label >
+        </label>
         <input
-        className='input-login'
+          className="input-login"
           data-testid="input-form-password"
           type="password"
           name="password"
           ref={register({ required: true })}
-         
         />
         {errors.password && "This field is required"}
-
-        <button  className='button-login' data-testid="submit-button" type="submit">
-          {" "}
-          Login{" "}
-        </button>
-      </form >
-      <button className='button-login'
-        onClick={() => (window.location = "http://localhost:5000/auth/google")}
-      >
-        {" "}
-        Google +
-      </button>
+        </div>
+        <div className='buttonsDiv-login'>
+          <button
+            className="button-login"
+            data-testid="submit-button"
+            type="submit"
+          >
+            {" "}
+            Login{" "}
+          </button>
+          <button
+            className="button-login"
+            onClick={() =>
+              (window.location = "http://localhost:5000/auth/google")
```

Comment:
Not sure if we spoke about this already (otherwise we probably should hehe). But this is very explicit set to localhost. This won't work when we deploy this project to an actual webserver which is behind a hostname. So in our next meeting let's talk a bit about configuring environments :) 

Not blocking for now or anything though :)

Reply:
I still could not solve the google oAuth situation. Good to check it later.